engine/src/main/java/org/operaton/bpm/engine/impl/persistence/entity/TaskEntity.java

```diff
@@ -178,33 +178,33 @@ public class TaskEntity extends AbstractVariableScope implements Task, DelegateT
 
   protected transient List<PropertyChange> identityLinkChanges = new ArrayList<>();
 
-  protected List<VariableInstanceLifecycleListener<VariableInstanceEntity>> customLifecycleListeners;
+  protected transient List<VariableInstanceLifecycleListener<VariableInstanceEntity>> customLifecycleListeners;
 
   // name references of tracked properties
-  public static final String ASSIGNEE = "assignee";
-  public static final String DELEGATION = "delegation";
-  public static final String DELETE = "delete";
-  public static final String DESCRIPTION = "description";
-  public static final String DUE_DATE = "dueDate";
-  public static final String FOLLOW_UP_DATE = "followUpDate";
-  public static final String NAME = "name";
-  public static final String OWNER = "owner";
-  public static final String PARENT_TASK = "parentTask";
-  public static final String PRIORITY = "priority";
-  public static final String CASE_INSTANCE_ID = "caseInstanceId";
+  public static final String PROPERTY_ASSIGNEE = "assignee";
```

Comment:
Unfortunately these fields are public. Even when the likelyness of any client using them is low, I would consider them as API for now.
It will be possible to rename them if we would have an OpenRewrite recipe to migrate usages. Please raise an issue to https://github.com/operaton/migrate-from-camunda-recipe describing the renaming, or revert.

Reply:
I've reverted the renaming of these fields